app/repos/repository_podio.rb

```diff
@@ -18,25 +18,80 @@ def change_status(id, application)
         'status-expa': map_status(application.exchange_participant.status.to_sym),
         'teste-di-data-do-applied': parse_date(application.applied_at),
         'teste-di-data-do-accepted': parse_date(application.accepted_at),
-        'teste-di-data-do-approved': parse_date(application.approved_at),
-        'teste-di-data-do-break-approval': parse_date(application.break_approved_at)
       }}
       item = Podio::Item.update(id, attrs)
       item
     end
 
+    def send_application(id, application, approved_sync_count)
+      check_podio
+
+      attrs = {'fields': {
+        "data-do-approved-#{approved_sync_count}": parse_date(application.approved_at),
+        "link-da-vaga-#{approved_sync_count}-tnid-#{approved_sync_count}": embed_id(application),
+        "produto-apd-#{approved_sync_count}": product_index(application),
+        "expa-application-id-#{approved_sync_count}": application.expa_id.to_s
+      }}
+
+      item = Podio::Item.update(id, attrs)
+
+      if item
+        update_approved_sync_count(application.exchange_participant)
+        update_application(application)
+      end
+
+      item
+    end
+
+    def update_approved_sync_count(exchange_participant)
+      exchange_participant.update_attributes(approved_sync_count: exchange_participant.reload.approved_sync_count + 1)
+      exchange_participant.reload
+    end
+
+    def update_application_podio_status(application)
+      check_podio
+      attrs = {'fields': {
+        'status-expa': map_status_prep(application.status.to_sym)
+      }}
+      item = Podio::Item.update(application.podio_id, attrs)
+      item
+    end
+
+    def update_application(application)
```

Comment:
These methods should be private, right? The name is could be more specific too

Reply:
Oh no... i understand. This repository, intended to be only for Podio. Not for Applications. 

ON my branch, i created an Application Repository.